### PR TITLE
Embed Validating Models 101 playlist into Guides section

### PR DESCRIPTION
**Pull request by nrichers**
## Internal Notes for Reviewers

This PR embeds the playlist for "Validating Models 101" into our Guides landing page. This playlist includes five short videos, re-recorded with higher production quality, that cover:

- Intro & quick tour for validators
- Review model documentation
- Analyze test results
- Prepare validation report
- Submit for review & approval

[**LIVE PREVIEW**](https://docs-demo.vm.validmind.ai/pr_previews/nrichers/sc-6631/re-record-validating-models-101-videos/guide/guides.html#model-validation)

image

 

## External Release Notes

We created a series of short videos that introduce you to validating models on our platform. [Watch the video series ...](../../guide/guides.html#model-validation)

### CODE REVIEW

File: site/guide/guides.qmd [hunk 1]
```diff
@@ -153,6 +153,8 @@ Set up validation guidelines and prepare validation reports, work with findings
 :::{#guides-model-validation}
 :::
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?si=JfBNClmsZYLS7m9F&amp;list=PL7I7ZrCoVCCObEiuLLAL0Wm9GQ6x5U0q5" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```
Comment (validbeck) [inline site/guide/guides.qmd:156]:
```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?si=JfBNClmsZYLS7m9F&amp;list=PL7I7ZrCoVCCObEiuLLAL0Wm9GQ6x5U0q5" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen style="box-shadow: 5px 5px 5px #ccc, -5px 5px 5px #ccc; border-radius: 5px;"></iframe>
```

File: site/guide/guides.qmd [hunk 2]
```diff
@@ -153,6 +153,8 @@ Set up validation guidelines and prepare validation reports, work with findings
 :::{#guides-model-validation}
 :::
```
Comment (validbeck) [inline site/guide/guides.qmd:155]:
```suggestion
Watch a full walkthrough on model validation: 
```

**Review by validbeck — APPROVED** [review]
Nice! Just making sure our videos follow the cleaned-up drop-shadow & rounded corner format here. Might be good to have some lead in text maybe?

File: site/guide/guides.qmd (hunk 2 above)
Comment (validbeck) [reply site/guide/guides.qmd:155]: @nrichers Can I ask why you rejected this change? For me, users don't usually want to click on videos unless they're clear on what value they bring.

Comment (nrichers) [reply site/guide/guides.qmd:155]: @validbeck two things, the original change breaks the video alignment but after looking at it in the preview, I don't think the text actually adds anything. I love the simple look of the listing tiles — the video just underneath those without any additional text looks very clean. Add the next, not so much.

image

Comment (validbeck) [reply site/guide/guides.qmd:155]: Yes, ideally there should be a line break and THEN the video — apologies for not adding that in the suggestion. Can we try that and see how it looks?

Comment (nrichers) [reply site/guide/guides.qmd:155]: You can but today's storm in a teacup is all about whether you need lead-in text at all: https://github.com/validmind/documentation/pull/450#issuecomment-2377559283

('Add lightness' would suggest you practice radical simplicity and save the words. Our docs site is already word heavy.)

**Comment by nrichers** [comment]
@mehdi0501 I'm going to merge this, but if you have feedback, please open a story or let me know what the feedback is and I will open a story to make sure it's addressed.

**Pull request merged**